cmd/kubeadm/app/util/apiclient/idempotency_test.go

```diff
@@ -237,54 +237,3 @@ func TestMutateConfigMapWithConflict(t *testing.T) {
 		t.Fatalf("ConfigMap mutation with conflict was invalid, has: %q", cm.Data["key"])
 	}
 }
-
-func TestGetConfigMapWithShortRetry(t *testing.T) {
```

Comment:
these unit tests no longer make sense. testing the internal context.Background() is not exactly feasible, but if someone has ideas for tests please send a follow up.